lib/helpers/error-redirect.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/**
+ * Use this method to render JSON error responses.
+ *
+ * @function
+ *
+ * @param {Object} res - Express http response.
+ * @param {Object} err - An error object.
+ */
+function errorRedirect(res, err) {
+  var message = 'Unknown error. Please contact support.';
+
+  if (err) {
```

Comment:
Copied this from the original jsonError function. I suppose it's possible to trigger it without an error object that includes a message?